spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -322,11 +332,30 @@ def get_bgcolor(self, index):
             color.setAlphaF(.3)
         return color
 
+    def update_search_letters(self, text):
+        """Update search letters with text input in search box."""
+        self.letters = text
+        names = self.keys
+        results = get_search_scores(text, names, template='<b>{0}</b>')
+        self.normal_text, self.rich_text, self.scores = zip(*results)

```

Comment:
Are we using the `rich_text` ? (could not see that on the gif)

Reply:
I will check :+1:

Reply:
@goanpeca  a preview with using the `rich_text`:
![imagen](https://user-images.githubusercontent.com/16781833/58226924-603a3980-7cee-11e9-862b-56e0d226490b.png)

Reply:
It looks nice, but leave the code somewhere while we can fix the HTMLDelegate height as we discussed on the meeting